### PR TITLE
Add hamming distance

### PR DESCRIPTION
**Pull request by duncan-clark**
Added compareNet to do faster comparison.
Also added some adhoc tests - need to have proper testing for stats soon, but will do that in another PR.
Where should terms be documented ?

### CODE REVIEW

File: tests/temp_adhoc_terms_testt.R [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+      Sys.sleep(1)
+    }
+  }
+}
+
+# =====================================================================
+# hamming Function
+# =====================================================================
+
+edges = as.edgelist(net)*1
```
Comment (ifellows) [inline tests/temp_adhoc_terms_testt.R:342]: These tests scenarios are good; however, we should throw an error (or use testthat) when the output is not as expected. For example, check that hamming_calc and calculateStatistics come up with the same values

**Review by ifellows — COMMENTED** [review]
not for this pull, but we should have terms documented as in https://github.com/statnet/lolog/blob/202b0aa9532762a919b8dc4af79072e867bd28ff/R/roxygen.R#L101

File: tests/temp_adhoc_terms_testt.R (hunk 1 above)
Comment (duncan-clark) [reply tests/temp_adhoc_terms_testt.R:342]: agree - I noticed there was a rejigging of the testing structure in LOLOG - should we go down that route too? I put these here mainly so you could see I did at least SOME testing, not as a long term testing workflow for additional statistics

Comment (ifellows) [reply tests/temp_adhoc_terms_testt.R:342]: yes, I'd recommend the lolog test structure. It is (I believe) the standard way to structure testing with testthat.

Comment (ifellows) [reply tests/temp_adhoc_terms_testt.R:342]: did you want to leave that for another pull?

Comment (duncan-clark) [reply tests/temp_adhoc_terms_testt.R:342]: yes please - if thats okay - i'll do it next then add more terms in the new testing framework

**Pull request merged**